docs/asset-store/11-04-minio.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Minio
+type: Metrics
+---
+
+As an external, open-source asset storage solution, Minio exposes its own metrics. See the [official documentation](https://github.com/minio/minio/tree/master/docs/metrics) for details.
```

Comment:
`asset storage` -> `file storage`

Reply:
I would add that AssetStores comes with preconfigured ServiceMonitor that scraps those metrics into prometheus, so you can already build your own grafana dashboard using them or reuse the existing one we prepared